src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -464,42 +485,97 @@ class TableListView<V extends {}> extends React.Component<
       filters: searchFilters ?? [],
     };
 
-    const columns = this.props.tableColumns.slice();
-    if (this.props.editItem) {
-      columns.push({
-        name: i18n.translate('kibana-react.tableListView.listing.table.actionTitle', {
-          defaultMessage: 'Actions',
-        }),
-        width: '100px',
-        actions,
-      });
-    }
-
     const noItemsMessage = (
       <FormattedMessage
         id="kibana-react.tableListView.listing.noMatchedItemsMessage"
         defaultMessage="No {entityNamePlural} matched your search."
         values={{ entityNamePlural: this.props.entityNamePlural }}
       />
     );
+
     return (
       <EuiInMemoryTable
         itemId="id"
         items={this.state.items}
-        columns={columns}
+        columns={this.getTableColumns()}
         pagination={this.pagination}
         loading={this.state.isFetchingItems}
         message={noItemsMessage}
         selection={selection}
         search={search}
-        sorting={true}
+        sorting={{ sort: this.state.tableSort as PropertySort }}
+        onChange={this.onTableChange.bind(this)}
         data-test-subj="itemsInMemTable"
         rowHeader={this.props.rowHeader}
         tableCaption={this.props.tableCaption}
       />
     );
   }
 
+  getTableColumns() {
+    const columns = this.props.tableColumns.slice();
+
+    // Add "Last update" column
+    if (this.state.hasUpdatedAtMetadata) {
+      const renderUpdatedAt = (dateTime?: string) => {
+        if (!dateTime) {
+          return <span />;
+        }
+        const updatedAt = moment(dateTime);
+
+        if (updatedAt.diff(moment(), 'days') > -7) {
+          return (
+            <FormattedRelative value={new Date(dateTime).getTime()}>
+              {(formattedDate: string) => <span>{formattedDate}</span>}
+            </FormattedRelative>
+          );
+        }
+        return <span>{updatedAt.format('L @ LT')}</span>;
+      };
+
+      columns.push({
+        field: 'updatedAt',
+        name: i18n.translate('kibana-react.tableListView.lastUpdatedColumnTitle', {
+          defaultMessage: 'Last updated',
+        }),
+        render: (field: string, record: { updatedAt?: string }) =>
+          renderUpdatedAt(record.updatedAt),
+        sortable: true,
```

Comment:
When there is no tags columns like in the Graph listing page (only 4 columns total) this column actually takes up too much space. Squishing the title and description unnecessarily. We should set a pixel-width so it doesn't grow too wide. I think `150px` should be enough.

```suggestion
        sortable: true,
        width: '150px',
```

Reply:
Great call 👍